lib/web/apiserver_test.go

```diff
@@ -3565,3 +3566,35 @@ type mockProxySettings struct {
 func (mock *mockProxySettings) GetProxySettings(ctx context.Context) (*webclient.ProxySettings, error) {
 	return &webclient.ProxySettings{}, nil
 }
+
+func TestIsRoleMatchError(t *testing.T) {
+	t.Parallel()
+	matches := []struct {
+		name string
+		err  error
+	}{
+		{"saml", trace.AccessDenied("unable to map attributes to role for connector: x")},
```

Comment:
nit: use keyed struct initialization

```suggestion
		{name: "saml", err: trace.AccessDenied("unable to map attributes to role for connector: x")},
```

same for others.